python/lsst/rubintv/production/processingControl.py

```diff
@@ -1275,6 +1275,18 @@ def setRaftOff(self, raftName: str) -> None:
             if detector.getName().startswith(raftName):
                 self._detectorStates[detector] = False
 
+    def setDetectorOn(self, detectorNumber: int) -> None:
+        # TODO: This really is grossly inefficient and could be costly
```

Comment:
A 189-element loop is actually pretty unlikely to be a bottleneck if this is all you're doing in it.  I don't think this needs a `TODO` even if it feels gross.